test/unit/accessibility/describe.js

```diff
@@ -0,0 +1,171 @@
+suite('describe', function() {
+  let myp5;
+  let myp5Container;
+  let myID = 'myCanvasID';
+  let a = 'a';
+  let b = 'b';
+  let c = 'c';
+
+  setup(function(done) {
+    myp5Container = document.createElement('div');
+    document.body.appendChild(myp5Container);
+    new p5(function(p) {
+      p.setup = function() {
+        let cnv = p.createCanvas(100, 100);
+        cnv.id(myID);
+        myp5 = p;
+        done();
+      };
+    }, myp5Container);
+  });
+
+  teardown(function() {
+    myp5.remove();
+    if (myp5Container && myp5Container.parentNode) {
+      myp5Container.parentNode.removeChild(myp5Container);
+    }
+    p5Container = null;
+  });
+
+  const getInner = elID => {
+    return document.getElementById(elID).innerHTML;
+  };
+
+  suite('p5.prototype.describe', function() {
+    let expected = 'a.';
+    test('should be a function', function() {
+      assert.ok(myp5.describe);
+      assert.typeOf(myp5.describe, 'function');
+    });
+    test('should create description as fallback', function() {
+      myp5.describe(a);
+      let actual = getInner(myID + '_fallbackDesc');
+      assert.deepEqual(actual, expected);
+    });
+    /*test('should not add extra period if string ends in "."', function() {
+      myp5.describe('a.');
+      let actual = getInner(myID + '_fallbackDesc');
+      assert.deepEqual(actual, expected);
+    });*/
```

Comment:
@kjhollen this is one of the tests that are not working. I've commented out for now. when I try doing what the test does on a p5 sketch it doesn't give me any errors so I have to think through it a bit and figure out what's going on. There are 3 more tests where that I've commented out because, like this one, they keep giving me unexpected errors. If you have any thoughts lmk 😅

Reply:
ah shoot! ok, I'm checking out the code now and will let you know if the behavior is the same for me.

Reply:
Ok, I've got a theory. I think it has to do with the way the template strings are being compiled down to ES5 with babel: it's including the indents and linebreaks. I think this is causing a mismatch sometimes.

More here: https://medium.com/dailyjs/how-to-create-multi-line-string-with-template-literals-in-javascript-a3a140d0b0f6

I think you can use a back slash to fix it at the end of each line: https://stackoverflow.com/questions/37321047/wrap-long-template-literal-line-to-multiline-without-creating-a-new-line-in-the

(trying this out now on my end!)

Reply:
ok, more complex than it looks! I can't get it to work with the minified p5.js file, but the regular one works fine. hmm. any thoughts @lmccart?